File: part3/functionalProgramming/write_to_file.erl
%%%-------------------------------------------------------------------
%%% @author xxx
%%% @copyright (C) 2022, <COMPANY>
%%% @doc
%%%
%%% @end
%%% Created : 18. 十二月 2022 16:56
%%%-------------------------------------------------------------------
-module(write_to_file).
-author("xxx").

%% API
-export([start/0]).
start() ->
%%  注意 - 要打开我们必须使用的文件，打开（文件名，模式）。
  {ok, File1} = file:open("Tempfile2.txt", [write]),
%% 将内容写入文件的语法：write（filemode，file_content）。
  file:write(File1,"Writting contents to file"),
  io:fwrite("Data inserted into file\n").

File: part3/functionalProgramming/write_to_file.cpp
#include <iostream>
#include <fstream>
// - fstream 是用于控制文件读/写操作的流类。
//- ofstream 是用于将内容写入文件的流类。
using namespace std;

int main () {
    ofstream myfile;
    myfile.open ("Tempfile.txt", ios::out);
    myfile << "Writing Contents to file.\n";
    cout << "Data inserted into file";
    myfile.close();
    return 0;
}
